docs/options.md

```diff
@@ -1334,14 +1336,21 @@ tree. To access your test code, you have a couple of options:
     - `{project}` is an absolute path to the project root - the working
       directory where cibuildwheel was called.
 
-On all platforms other than iOS, the command is run in a shell, so you can write things like `cmd1 && cmd2`.
+    These placeholders are not available on Android and iOS, because those
+    platforms run tests in a virtual machine that does not have access to
+    the build machine's filesystem.
 
-On iOS, the value of the `test-command` setting must follow the format `python
--m MODULE [ARGS...]` - where MODULE is a Python module name, followed by
-arguments that will be assigned to `sys.argv`. Other commands cannot be used.
+On all platforms other than Android and iOS, the command is run in a shell, so you can write things like `cmd1 && cmd2`.
+
+On Android and iOS, the command is parsed by `shlex.split`, and is required to
+be in one of the following forms:
+
+* `python -c command ...` (Android only)
+* `python -m module-name ...`
+* `pytest ...` (deprecated; converted to `python -m pytest ...`)
```

Comment:
I'd rather just not document this form, I think it's more of a convenience feature for users migrating from other platforms.

```suggestion
```

Reply:
Done.